examples/create_server_profile_boot_from_SAN.yml

```diff
@@ -0,0 +1,115 @@
+###
+# Copyright 2017 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+- hosts: all
+  vars:
+    config: '{{ playbook_dir }}/oneview_config.json'
+    server_hardware_type_name: "SY 480 Gen9 3"
+    enclosure_group_name: "SYN107_EG"
+    server_hardware_name: "2-USE722C99W-M, bay 1"
+    profile_name: "profile_san"
+    management_network: "Management"
+    SAN_A_network: "SAN-A"
+    SAN_B_network: "SAN-B"
+    storage_pool_name_A: "Pool name A"
+
+  tasks:
+
+    - name : "Create Server Profile - {{ profile_name }}"
+      oneview_server_profile:
+        config: "{{ config }}"
+        data:
+          name: "{{ profile_name }}"
+          serverHardwareTypeName: "{{ server_hardware_type_name }}"
+          enclosureGroupName: "{{ enclosure_group_name }}"
+          server_hardware: "{{ server_hardware_name }}"
```

Comment:
This will not work anymore as server_hardware was not following the style guide from other fields. It should be 'serverHardwareName' now.

Reply:
Fixed.